lib/firebase/firebase_facebook_helper.rb

```diff
@@ -7,16 +7,17 @@ def self.open_facebook_session(options={}, &block)
 
   def open_facebook_session(options={}, &block)
     ref = self
-    permissions = options[:permissions] || ['public_profile']
-    allow_ui = options.fetch(:allow_login_ui, true)
-    FBSession.openActiveSessionWithReadPermissions(permissions, allowLoginUI: allow_ui,
-      completionHandler: -> (session, state, error) do
-        if error
-          block.call(error, nil)
-        elsif state == FBSessionStateOpen
-          token = session.accessTokenData.accessToken
+    fb_login = FBSDKLoginManager.alloc.init
+    fb_login.logInWithReadPermissions(["email"], 
```

Comment:
Is there something wrong with using the `:permissions` option, instead of hard coding it?